lib/Doctrine/ORM/EntityManager.php

```diff
@@ -222,12 +222,8 @@ public function getCache()
     /**
      * {@inheritDoc}
      */
-    public function transactional($func)
+    public function transactional(callable $func)
```

Comment:
@Majkl578 It seems this was possible even with older versions. https://3v4l.org/I7d3P